docs/how-to-guides/how-to-run-chainhook-as-a-service-using-stacks.md

```diff
@@ -48,31 +48,43 @@ events_keys = ["*"]
 In this section, you will configure a chainhook to communicate with the network. Run the following command in your terminal and generate the `Chainhook.toml` file.
 
 ```console
-chainhook config generate --testnet
+chainhook config generate --mainnet
 ```
 
-Ensure that the `bitcoind_rpc_url`, `bitcoind_rpc_username`, `bitcoind_rpc_password` match with the `rpcport`, `rpcuser` and `rpcpassword` in the `bitcoin.conf` file and the port of the `stacks_node_rpc_url` matches the `rpc_bind` in the `Stacks.toml` file.
+Several network parameters in the generated `Chainhook.toml` configuration file need to match those in the `bitcoin.conf` file created earlier in the [Setting up a Bitcoin Node](#setting-up-a-bitcoin-node) section. Please update the following parameters accordingly:
+
+1. Update `bitcoind_rpc_username` with the username set for `rpcuser` in `bitcoin.conf`.
+2. Update `bitcoind_rpc_password` with the password set for `rpcpassword` in `bitcoin.conf`.
+3. Update `bitcoind_rpc_url` with the same host and port used for `rpcport` in `bitcoin.conf`.
+4. Ensure `stacks_node_rpc_url` matches the `rpc_bind` in the `Stacks.toml`.
 
 The following `Chainhook.toml` file is generated:
 
 ```toml
 [storage]
 working_dir = "cache"
 
-# The HTTP API allows you to register/deregister
-# dynamically predicates
-# Disable by default
+# The Http Api allows you to register / deregister
+# dynamically predicates.
+# Disable by default.
 #
 # [http_api]
 # http_port = 20456
 # database_uri = "redis://localhost:6379/"
 
 [network]
 mode = "mainnet"
-bitcoind_rpc_url = "http://localhost:8332"                # Must match with the rpcport in the bitcoin.conf
-bitcoind_rpc_username = "<bitcoind_username>"             # Must match with the rpcuser in the bitcoin.conf
-bitcoind_rpc_password = "<bitcoind_password>"             # Must match with the rpcpassword in the bitcoin.conf
-stacks_node_rpc_url = "http://localhost:20443"            # Must match with the rpc_bind in the Stacks.toml file
+bitcoind_rpc_url = "http://localhost:8332"
+bitcoind_rpc_username = "devnet"
+bitcoind_rpc_password = "devnet"
+# Bitcoin block events can be received by Chainhook
+# either through a Bitcoin node's ZeroMQ interface,
+# or through the Stacks node. The Stacks node is
+# used by default:
+stacks_node_rpc_url = "http://localhost:20443"
+stacks_events_ingestion_port = 20455
```

Comment:
this configuration was mentioned in the previous version of the docs, but was never present in the `Chainhook.toml`

i was also unable to reproduce any meaningful differences by including or excluding, so would love to get more clarity around the `stacks_events_ingestion_port`

Reply:
I think this is something @MicaiahReid could comment on

Reply:
Indeed, Micaiah spoke to it below, see comment: https://github.com/hirosystems/chainhook/pull/384#discussion_r1296005638

Reply:
@ryanwaits — is there anything you think should be added to the Chainhook.toml generator [PR](https://github.com/hirosystems/chainhook/pull/382) ([link to generator.rs file](https://github.com/hirosystems/chainhook/blob/improvement/cli-config-generator/components/chainhook-cli/src/config/generator.rs))to add clarity to this?